site/docs/components/card/examples.mdx

```diff
@@ -10,46 +10,57 @@ data:
 
 <LivePreviewControls>
 
-    <LivePreview componentName="card" exampleName="Default" >
-      ### Default
-    </LivePreview>
+    <LivePreview componentName="card" exampleName="Default" displayName="Card" >
+      ### Card
 
-    <LivePreview componentName="card" exampleName="Secondary" >
-      ### Secondary
+      By default, the Card component does not have any interactive or focusable elements within it. 
     </LivePreview>
 
-    <LivePreview componentName="card" exampleName="Actions" >
-      ### Adding actions
+        <LivePreview componentName="card" exampleName="Variants" >
+      ### Variants
 
-      If you intend to add any links or buttons within your card, use a default card rather than interactable card to ensure the component is accessible.
+      All card components provide both primary and secondary variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
     </LivePreview>
 
-    <LivePreview componentName="card" exampleName="InteractableCard" >
-      ### Interactable card
 
-      - Use the `InteractableCard` component to make the entire card interactable. Don't use any focusable elements within interactable cards.
-      - To use as a link, wrap `Card` in [`Link`](../link).
-      - Avoid adding too much content to interactable cards. All users should be able to read them easily, and it shouldn't take too long for a screen reader tool to read out the content.
-    </LivePreview>
+    <LivePreview componentName="card" exampleName="Accents" >
+      ### Accents
+
+    - The accent in a Card is optional, however, it can be used to indicate meaning (such as Product or Brand color).
+    - An accent can work well with the hoverable prop, to indicate it is interactive when the user's focus is over a specific card.
+    - Maintain consistency in accent placement across cards to ensure a coherent visual experience in your UI.
 
-    <LivePreview componentName="card" exampleName="SecondaryInteractableCard" >
-      ### Secondary Interactable Card
     </LivePreview>
 
-    <LivePreview componentName="card" exampleName="DisabledInteractableCard" >
-      ### Disabled interactable card
-      - Set `disabled={true}` to apply disabled styling to interactable cards.
-      - Make sure to also apply `disabled={true}` to any nested components, such as `Text`, to ensure disabled styling also applies to the content.
+    <LivePreview componentName="card" exampleName="Actions" >
+      ### Actions
+
+    Use the Card to incorporate actions like [links](../link) or [buttons](../button) within it. Avoid embedding actions into a Link card or an Interactable card.
```

Comment:
cc @mikearildbrown 

This sentence seems inconsistent with what we do elsewhere and even in Card's usage guidance e.g.

```mdx
Link cards shouldn't contain actionable components (like [Buttons](../button) or [Links](../link)), as the card itself is the interactive element.
```